Form-Controls/index.html

```diff
@@ -4,25 +4,53 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <title>My form exercise</title>
-  <meta name="description" content="">
+  <meta name="description" content="T-shirt order form">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <header>
+  <header role="banner">
     <h1>Product Pick</h1>
   </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <main role="main" aria-labelledby="form-title">
+    <form id="tshirt-order" aria-labelledby="order-title">
+      <h2 id="order-title">Order your T-shirt</h2>
+      
+      <label for="name">Name:</label>
+      <input type="text" id="name" name="name" required pattern="[A-Za-z\s]{2,}" title="Please enter at least 2 characters" aria-label="Name input field">
+      
+      <label for="email">Email:</label>
+      <input type="email" id="email" name="email" required aria-describedby="email-error" />
+      <span id="email-error" class="visually-hidden">Please enter a valid email address (e.g., example@domain.com).</span>
 
+      <label for="color">T-Shirt Color:</label>
+      <select name="color" id="color" required aria-label="Select T-shirt color">
+        <option value="">Select a color</option>
+        <option value="red">Red</option>
+        <option value="blue">Blue</option>
+        <option value="green">Green</option>
+      </select>
+      
+      <label for="size">T-shirt Size:</label>
+      <select name="size" id="size" required aria-label="Select T-shirt size">
```

Comment:
Again, I love that you're thinking about ARIA! ❤️ ⭐ 

Bad ARIA is probably worse than no ARIA, however. Don't use accessibility features to re-state existing features. In this case, the html tag "label" is labelling the select input already, so you don't need to label it again with ARIA.